pkg/ccl/backupccl/backup_processor.go

```diff
@@ -268,14 +253,8 @@ func runBackupProcessor(
 						StartTime:                           span.start,
 						EnableTimeBoundIteratorOptimization: useTBI.Get(&clusterSettings.SV),
 						MVCCFilter:                          spec.MVCCFilter,
-						ReturnSstBelowSize:                  smallFileSize.Get(&clusterSettings.SV),
 						TargetFileSize:                      storageccl.ExportRequestTargetFileSize.Get(&clusterSettings.SV),
-					}
-					if writeSSTsInProcessor {
-						req.ReturnSST = true
-					} else {
-						req.Storage = defaultConf
-						req.Encryption = spec.Encryption
+						ReturnSST:                           true,
```

Comment:
Can we simplify the condition below, as well as the comment on top of it (L304, but GH won't let me comment there):

```
if req.ReturnSST {
						header.TargetBytes = 1
}
```